suites/quincy/rados/tier-2_rados_test_bluestore.yaml

```diff
@@ -0,0 +1,127 @@
+# Suite contains basic tier-2 rados tests
```

Comment:
As discussed, please add the test to existing tier-2 test suites, instead creating a new one.